website/docs/components/models/openai.md

```diff
@@ -7,26 +7,72 @@ sidebar_position: 1
 
 To use a language model hosted on OpenAI (or compatible), specify the `openai` path in the `from` field.
 
-For a specific model, include it as the model ID in the `from` field (see example below). The default model is `gpt-3.5-turbo`.
+For a specific model, include it as the model ID in the `from` field (see example below). The default model is `gpt-4o-mini`.
 
-These parameters are specific to OpenAI models:
+```yaml
+models:
+  - from: openai:gpt-4o-mini
+    name: openai_model
+    params:
+      openai_api_key: ${ secrets:OPENAI_API_KEY } # Required for official OpenAI models
+      tools: auto # Optional. Connect the model to datasets via SQL query/vector search tools
+      system_prompt: "You are a helpful assistant." # Optional.
+
+      # Optional parameters
+      endpoint: https://api.openai.com/v1 # Override to use a compatible provider (i.e. NVidia NIM)
+      openai_org_id: ${ secrets:OPENAI_ORG_ID }
+      openai_project_id: ${ secrets:OPENAI_PROJECT_ID }
+
+      # Override default chat completion request parameters
+      openai_temperature: 0.1
+      openai_response_format: { "type": "json_object" }
+```
+
+## Configuration
+
+### `from`
+
+The `from` field takes the form `openai:model_id` where `model_id` is the model ID of the OpenAI model, valid model IDs are found in the `{endpoint}/v1/models` API response.
+
+Example:
+
+```bash
+curl -H "Authorization: Bearer $OPENAI_API_KEY" https://api.openai.com/v1/models
+```
+
+```json
+{
+  "object": "list",
+  "data": [
+    {
+      "id": "gpt-4o-mini",
+      "object": "model",
+      "created": 1727389042,
+      "owned_by": "system"
+    },
+...
+}
+```
+
+### `name`
+
+The model name. This will be used as the model ID within Spice and Spice's endpoints (i.e. `http://localhost:8090/v1/models`). This can be set to the same value as the model ID in the `from` field.
+
+### `params`
 
 | Param               | Description                   | Default                     |
 | ------------------- | ----------------------------- | --------------------------- |
+| `endpoint`          | The OpenAI API base endpoint. Can be overridden to use a compatible provider (i.e. NVidia NIM). | `https://api.openai.com/v1` |
+| `tools`             | Which [tools] should be made available to the model. Set to `auto` to use all available tools. | -    |
```

Comment:
`[tools]` missing link?

Reply:
It is a "reference-style links" in Markdown. It allows you to keep your text cleaner by separating the link URLs from the content. There are two parts to it:

1. The link reference in your text: `[text][reference]` or `[text][]` (if the reference matches the text)
2. The link definition anywhere else in the document: `[reference]: URL`

For example:
```markdown
Here's a [link][example] to a website.
Check out [Google][] too.

[example]: https://example.com
[Google]: https://google.com
```

This is particularly useful when you have multiple links to the same URL or when you want to keep your main text more readable by collecting all the URLs in one place.